android/src/main/java/cz/ackee/extensions/android/DrawableExtensions.kt

```diff
@@ -107,3 +106,24 @@ fun colorStateList(
     ).filterValues { it != null }
     return ColorStateList(states.keys.toTypedArray(), states.values.map { it!!.toInt() }.toIntArray())
 }
+
+/**
+ * Convert this Drawable to Bitmap representation. Should take care of every Drawable type
+ */
+fun Drawable.toBitmap(): Bitmap {
```

Comment:
isnt this breaking change?

Reply:
Yes it is, but it doesn't make sense to have this in `ViewExtensions`, it clearly belongs to `DrawableExtensions`.

Reply:
I agree with you but I dont know if you rememeber how people were pissed about the last breaking change (and I understand them) and now you want to introduce another? :D Btw is it really breaking change? because i think that import for extensions does not contain file name ,eg 
```
import cz.ackee.extensions.android.backgroundDrawable
```
and since you are in the same package it should not affect target code in any way

Reply:
aha ok... Moreover idk if somebody uses this at all